src/ui/public/vislib/__tests__/visualizations/line_chart.js

```diff
@@ -149,21 +149,6 @@ describe('Vislib Line Chart', function () {
         });
       });
 
-      describe('containerTooSmall error', function () {
```

Comment:
I'm not too familiar with these tests, but why are we removing the tests? the container-too-small error is still valid to occur when making the visualization small